src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -342,14 +349,13 @@ public virtual void GenerateAnonymousFunctionConversionError(DiagnosticBag diagn
         // Returns the inferred return type, or null if none can be inferred.
         public BoundLambda Bind(NamedTypeSymbol delegateType)
         {
-            BoundLambda result;
-            if (!_bindingCache.TryGetValue(delegateType, out result))
+            if (_bindingCache.TryGetValue(delegateType, out var result))
```

Comment:
>var [](start = 60, length = 3)

Please spell out the type, or better revert the inlining of the declaration.

Reply:
:memo: According to the .editorconfig for this section of the code, both forms are acceptable. We need to update the configuration to match the practices followed during the code review process.

Reply:
➡️ Reverted style changes to the extent possible.

Reply:
> We need to update the configuration to match the practices followed during the code review process.

Note that the exact rules in compilers - ```var``` is not allowed unless the type is explicitly named in the initializer.